### PR TITLE
Create CrashHeadMod.java

### LINKED ISSUE #2054: CrashHead Exploit
The exploit who kick people from server like on this video: www.youtube.com/watch?v=LxAKwzNy-78

### PR DESCRIPTION
**Pull request by mcccs**
Fix #2054 
Not tested.

### CODE REVIEW

File: Wurst Client/src/tk/wurst_client/mods/CrashHeadMod.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+				WurstClient.INSTANCE.chat.error("Please take off your pants.");
+			setEnabled(false);
+			return;
+		}else if(!Minecraft.getMinecraft().thePlayer.capabilities.isCreativeMode)
+		{
+			WurstClient.INSTANCE.chat.error("Creative mode only.");
+			setEnabled(false);
+			return;
+		}
+		ItemStack stack = new ItemStack(Blocks.Skull);
```
Comment (mcccs) [inline Wurst Client/src/tk/wurst_client/mods/CrashHeadMod.java]: im not sure Blocks.Head or Blocks.Skull

**Comment by mcccs** [comment]
Idea: Mix this, #1814 and #1954 ,then call 2.4 "The Exploit Update"

File: Wurst Client/src/tk/wurst_client/mods/CrashHeadMod.java (hunk 1 above)
Comment (EnderGamer21) [inline Wurst Client/src/tk/wurst_client/mods/CrashHeadMod.java]: u can get yeads with a cmd if this is whats it is about

**Comment by mcccs** [comment]
Wait, this is not yet finished, you need to implement #1814 for this.

**Comment by Eyremba** [comment]
@mcccs can you please tell me the ".give" command for this crash-head?

**Comment by mcccs** [comment]
idont know, i only copied crashchest.

**Comment by Garkolym2** [comment]
mcccs commented 3 days ago
idont know, i only copied crashchest.
----------------------------------------------

Du scheiß Idiot, kannst auch nur in den GarPloit Source schauen. Peinlich, du bist der letzte abschauen xDDDDDD

**Comment by mcccs** [comment]
give someone Minecraft:SKull

**Comment by Eyremba** [comment]
@mcccs you can NOT use the minecraft:skull with the same NBT-Data like crashchest..... thats nooby and will not work. The NBT-Tags in the video with the HeadSkull Crasher are completely others.

**Comment by Garkolym2** [comment]
@Eyremba He is too stupid, to understand this. He need the sourcecode of the client

**Comment by 1337HaxorKiddie** [comment]
.skull should be right @mcccs

**Comment by 43ee3g** [comment]
mind sharing a way to prevent yourself from crashing?

**Comment by 1337HaxorKiddie** [comment]
@Eyremba Your right, with the Crash Chest nbt tags it wont work.

**Comment by mcccs** [comment]
Isaid #1814

**Comment by Nitrogen-Dioxide** [comment]
@mcccs 

It is minecraft:skull

**Comment by solidshadow96** [comment]
REMOVED O_O WHY

**Comment by 1337HaxorKiddie** [comment]
I think he removed it because it doesn´t work. @mcccs used the Crash Chest NBT tags for this item here, but that won´t work. These exploits are totaly different!

**Comment by solidshadow96** [comment]
He removed all mods that were in the milestone if you can call it that nothing was even added

**Pull request closed without merging**